File: POO JAVA/Aula09/src/aula09/Pessoa.java
package aula09;

public class Pessoa {
    
    //Atributos da Classe
    private String nome;
    private int idade;
    private String sexo;
    
    //Metodo Construtor

    public Pessoa(String nome, int idade, String sexo) {
        this.nome = nome;
        this.idade = idade;
        this.sexo = sexo;
    }
    
    //Metodos Acessores e Modificadores da Classe
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public int getIdade() {
        return idade;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }
    public String getSexo() {
        return sexo;
    }
    public void setSexo(String sexo) {
        this.sexo = sexo;
    }
    
    //Metodo da Classe
    public void fazerAniver(){
        this.idade++;
    }
}

File: POO PHP/Aula09/Pessoa.php
<?php

class Pessoa {

    //Atributos da Classe
    private $nome;
    private $idade;
    private $sexo;
    
    //Metodo Construtor
    public function __construct($nome, $idade, $sexo) {
        $this->nome = $nome;
        $this->idade = $idade;
        $this->sexo = $sexo;
    }

    //Metodos Acessores e Modificadores
    public function getNome() {
        return $this->nome;
    }
    public function getIdade() {
        return $this->idade;
    }
    public function getSexo() {
        return $this->sexo;
    }
    public function setNome($nome): void {
        $this->nome = $nome;
    }
    public function setIdade($idade): void {
        $this->idade = $idade;
    }
    public function setSexo($sexo): void {
        $this->sexo = $sexo;
    }
        
    //Metodo da Classe
    public function fazerAniversario() {
        $this->idade++;
    }
}
